hardware/PhilipsHue/PhilipsHue.cpp

```diff
@@ -936,14 +943,32 @@ void CPhilipsHue::ReportTemperature(const int NodeID, const float temperature, c
 	tsen.TEMP.id1 = (sID & 0xFF00) >> 8;
```

Comment:
could you please remove this, and use the default way of reporting a (temperature) sensor ?
Just call

`void SendTempSensor(const int NodeID, const int BatteryLevel, const float temperature, const std::string &defaultname);
`

These functions are specified in DomoticzHardware.cpp/h